nano/node/bootstrap.cpp

```diff
@@ -277,13 +281,13 @@ void nano::frontier_req_client::received_frontier (boost::system::error_code con
 {
 	if (!ec)
 	{
-		assert (size_a == sizeof (nano::uint256_union) + sizeof (nano::uint256_union));
+		assert (size_a == nano::frontier_req_client::size_frontier);
 		nano::account account;
-		nano::bufferstream account_stream (connection->receive_buffer->data (), sizeof (nano::uint256_union));
+		nano::bufferstream account_stream (connection->receive_buffer->data (), sizeof (nano::account));
```

Comment:
Why not change this to `sizeof(account)` if we're going to tie it to a type whose name doesn't express its on-wire representation ?

Reply:
Agreed, will fix